### PR TITLE
added templatable genesis config

### PR DESCRIPTION
**Pull request by BatiGencho**

### CODE REVIEW

File: chain.template.toml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+name="botanix_testnet"
+
+# >>>>>>>>>>> federation members secret keys
```
Comment (arminsabouri) [inline chain.template.toml]:
```suggestion
# >>>>>>>>>>> federation members public keys
```

Comment (BatiGencho) [reply chain.template.toml]: fixed

File: crates/node-core/src/args/genesis_args.rs [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    /// The account code (if any)
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub(crate) code: Option<String>,
+}
+
+#[derive(Clone, Debug, Deserialize)]
+#[serde(deny_unknown_fields, rename_all = "kebab-case")]
+pub(crate) struct FedMemberPubKey {
+    /// The pub key of the member
+    pub(crate) key: String,
```
Comment (arminsabouri) [inline crates/node-core/src/args/genesis_args.rs:37]: Super Nit:

```suggestion
    pub(crate) public_key: String,
```

**Review by arminsabouri — APPROVED** [review]
awesome stuff

**Pull request merged**